cmd/pdcli/alerts/cmd.go

```diff
@@ -0,0 +1,298 @@
+/*
+Copyright © 2021 Red Hat, Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alerts
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/olekukonko/tablewriter"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/constants"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/pdcli"
+	"github.com/spf13/cobra"
+)
+
+type alert struct {
+	incidentID string
+	name       string
+	clusterID  string
+	severity   string
+	status     string
+}
+
+var args struct {
+	high       bool
+	low        bool
+	assignment string
+	columns    string
+}
+
+var Cmd = &cobra.Command{
+	Use:   "alerts",
+	Short: "This command will list all the open alerts. \nIt will list all open high alerts assigned to self by default.",
+	RunE:  alertsHandler,
+}
+
+func init() {
+
+	// Urgency
+	Cmd.Flags().BoolVar(&args.low, "low", false, "view all low alerts")
+	Cmd.Flags().BoolVar(&args.high, "high", true, "view all high alerts")
+
+	// Incident Assignment
+	Cmd.Flags().StringVar(
+		&args.assignment,
+		"assigned-to",
+		"self",
+		"view alerts assigned to either self|team|silentTest",
+	)
+
+	// Columns displayed
+	Cmd.Flags().StringVar(
+		&args.columns,
+		"columns",
+		"incident.id,name,cluster.id,status,severity",
+		"Specify which columns to display separated by commas without any space in between",
+	)
+}
+
+// alertsHandler is the main alerts command handler.
+func alertsHandler(cmd *cobra.Command, args []string) error {
+	var incidentAlerts []alert
+	var alerts []alert
+
+	// Establish a secure connection with the PagerDuty API
+	connection, err := pdcli.NewConnection().Build()
+
+	if err != nil {
+		return err
+	}
+
+	// Get all incident IDs
+	incidentIDs, err := getIncidents(connection, true)
```

Comment:
should we be passing bool or a var like `severity` ?

Reply:
That really wasn't necessary there, removed it. 😃